csharp-101/shorturl_bulk.csv

```diff
@@ -0,0 +1,16 @@
+URL,Domain,Alias
```

Comment:
@katiesavage  @jamesmontemagno - this is the file used to bulk-add the tiny urls.